base_geoengine/fields.py

```diff
@@ -25,27 +28,140 @@ class GeoField(Field):
 
     geo_type = None
```

Comment:
`geo_type` includes the postgis type for the field when using "AddGeometryColumn".

I don't think this value is suited to `type` which is kind of odoo name for a type of field.

This also allow to use type to check if it has a prefix in 
https://github.com/OCA/geospatial/pull/123/files#diff-758bcd153189712715dfd56ddc0c293dR42

ok an `isinstance()` would do the trick but this makes the type more readable, ie. `type='geo_line'` has more meaning than `type='linestring'`